t/124-init-worker.t

```diff
@@ -756,3 +770,24 @@ GET /t
 ok
 --- no_error_log
 [error]
+
+
+
+=== TEST 20: destory Lua VM in cache processes
+--- skip_eval
```

Comment:
We should not skip this in valgrind test mode since this test case passes successfully in the HUP reload + valgrind combined test mode. It's fine for this test case to fail in the sole valgrind mode since our EC2 test cluster does support whitelist of expected failures in certain test modes or architectures:

https://github.com/openresty/opsboy/blob/master/misc/logs/parse-logs#L19